### PR TITLE
feat(): add support for new logging service

### PR DESCRIPTION
**Pull request by antmat**
Support for controlling new logging service via cocaine-tools

### CODE REVIEW

File: cocaine/tools/actions/logging.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    def __init__(self, logging_service):
+        self.logging_service = logging_service
+
+    @gen.coroutine
+    def execute(self):
+        raise NotImplementedError
+
+
+class LoggingConfigListLoggers(LoggingConfigurator):
+    def __init__(self, logging_service):
```
Comment (3Hren) [inline cocaine/tools/actions/logging.py]: You don't need to reimplement init if it does nothing except delegation

File: cocaine/tools/actions/logging.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+
+class LoggingConfigListFilters(LoggingConfigurator):
+    def __init__(self, logging_service):
+        super(LoggingConfigListFilters, self).__init__(logging_service)
+
+    @gen.coroutine
+    def execute(self):
+        filters = yield (yield self.logging_service.list_filters()).rx.get()
+        ret = {}
```
Comment (3Hren) [inline cocaine/tools/actions/logging.py]: Defaultdict?

File: cocaine/tools/actions/logging.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        for f in filters:
```
Comment (3Hren) [inline cocaine/tools/actions/logging.py]: Unpack values here because otherwise it's hard to understand what's going on

**Comment by antmat** [comment]
@noxiouz , @3Hren  PTAL

File: cocaine/tools/actions/logging.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    def execute(self):
+        filters = yield self.logging_service.remove_filter(self.filter_id)
+        raise gen.Return(filters)
+
+
+class LoggingConfigListFilters(LoggingConfigurator):
+    @gen.coroutine
+    def execute(self):
+        filters = yield (yield self.logging_service.list_filters()).rx.get()
+        ret = defaultdict(lambda: [])
```
Comment (noxiouz) [inline cocaine/tools/actions/logging.py]: defaultdict(list)

File: cocaine/tools/actions/logging.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        for (logger_name, filter_def, id, deadline) in filters:
```
Comment (noxiouz) [inline cocaine/tools/actions/logging.py]: () is not needed

**Comment by 3Hren** [comment]
LGTM

**Comment by noxiouz** [comment]
LGTM ~~split and squash~~

**Comment by antmat** [comment]
НБМХ

**Pull request merged**